app/views/trainees/_action_bar.html.erb

```diff
@@ -1,12 +1,16 @@
 <% unless paginated_trainees.empty? %>
   <%= render SortLinks::View.new %>
   <% if FeatureService.enabled?(:trainee_export) && policy(:trainee).export? %>
-    <div class="app-export--link">
-      <%= govuk_link_to(
-        I18n.t("views.trainees.index.export"),
-        export_results_path,
-        class: "app-trainee-export",
-      ) %>
-    </div>
+    <% if exceeds_export_limit?(@current_user, filtered_trainees.count) %>
+      <p class="text--right-align govuk-body govuk-hint govuk-!-margin-bottom-3">Admins cannot export more than 100 trainees</p>
```

Comment:
I reckon you could reuse the same `app-export--link` class or else put this in a similar div and not need the `text--right-align`. For both the link and this text, we essentially want the same positioning.

Reply:
For neatness the class could be renamed more generically `app-export--container` or similar.

Reply:
cheers - I'll have a look at that and see what it looks like